Model/JWTRequest.php

```diff
@@ -72,7 +72,12 @@ private function buildAuthHeader($method, $restUrl)
 
     private function buildURL($restUrl)
     {
-        return $this->tenant->getBaseUrl().$restUrl;
+        // Jira return absolute self links, so its more easy to work with get with absolute urls in such cases  
```

Comment:
Could you please describe the use-case and the difference in the logic for such a use-case?